### ISSUE TITLE
getNFTs with contractAddresses filter does not work

### ISSUE THREAD

**Issue by outdoteth** [alchemyplatform/alchemy-web3#89]
It fails to filter out the other addresses. For example on Ropsten:

```javascript
const nfts = await alchemy.getNfts({
  owner: "0xc9a6EC4aE16AD5A1B75Cf8f111FBD69eC3788774",
  contractAddresses: ["0x18e132bE028b77e6bb9bE98620bB335005077907"],
});
console.log("nfts", nfts);
```

Response includes addresses other than "0x18e132bE028b77e6bb9bE98620bB335005077907": 
![Screenshot 2022-02-06 at 14 24 32](https://user-images.githubusercontent.com/37438950/152685474-29558c32-a583-41ab-a5d4-af1b2dac80e1.png)

On closer inspection of the network tab, the library is sending the following params:

```
/v1/getNFTs/?owner=0xc9a6EC4aE16AD5A1B75Cf8f111FBD69eC3788774&contractAddresses=0x18e132bE028b77e6bb9bE98620bB335005077907
```

But this differs from the example in the documentation which says it should be:

```
/v1/getNFTs/?owner=0xc9a6EC4aE16AD5A1B75Cf8f111FBD69eC3788774&contractAddresses[]=0x18e132bE028b77e6bb9bE98620bB335005077907
```

(Note the additional `[]` after contractAddresses).

After using the format in the documentation, I get the expected result:

![Screenshot 2022-02-06 at 14 27 32](https://user-images.githubusercontent.com/37438950/152685574-307be6cb-2e7a-4d1f-a165-8e343e47fd3a.png)

**Comment by outdoteth** [comment]
Hmm, so I haven't confirmed this but I suspect this issue was introduced in this PR: 
https://github.com/alchemyplatform/alchemy-web3/pull/82

See this issue here: https://github.com/medialize/URI.js/issues/3

If I change it to be this then it works:

```javascript
const nfts = await alchemy.getNfts({
  owner: "0xc9a6EC4aE16AD5A1B75Cf8f111FBD69eC3788774",
  "contractAddresses[]": ["0x18e132bE028b77e6bb9bE98620bB335005077907"],
});
console.log("nfts", nfts);
```

![Screenshot 2022-02-06 at 20 12 30](https://user-images.githubusercontent.com/37438950/152699545-5e81fa7d-c500-4be1-8f11-1316a53250f3.png)

**Comment by dphilipson** [comment]
Hi there, thank you for the report! I am working on a fix.

**Comment by dphilipson** [comment]
Fix published in 1.2.2. Thank you very much for the report and sorry for the trouble!
